src/Node/index.js

```diff
@@ -114,6 +114,7 @@ export default class Node extends React.Component {
           textAnchor={textLayout.textAnchor}
           x={textLayout.x}
           y={textLayout.y}
+          transform={textLayout.transform}
```

Comment:
Please add `transform={textLayout.transform}` to the `<text>` element below this one (`"nodeAttributesBase"`) so that any attributes that are rendered are transformed along with the node title.